server/models/dockerimagemodel.py

```diff
@@ -0,0 +1,260 @@
+
+from docker import Client
+from docker.errors import DockerException
+from girder.constants import AccessType
+from girder.api.rest import getCurrentUser
+from girder.models.model_base import ModelImporter, AccessControlledModel
+
+import jsonschema
+
+from ..models import DockerImage, DockerImageError, \
+    DockerImageNotFoundError, DockerCache, DockerImageStructure
+
+# from six import iteritems
+# import os
+# from lxml import etree
+# from StringIO import StringIO
+
+
+class Dockerimagemodel(AccessControlledModel):
```

Comment:
Girder uses a function called `camelcase` to convert between a model name and the model's class name.  If the model name if `dockerimagemodel`. the class has to be `Dockerimagemodel`.  If, instead, the model were renamed to `docker_image_model`, then the class name is `DockerImageModel`.  To rename the class, you also have to rename the model.